src/App.js

```diff
@@ -46,6 +46,10 @@ const App = () => {
     }
   }
 
+  const getProfileLink = ()=>{
+      return <a href={`https://github.com/${login}`} rel="noopener noreferrer" target="_blank" >{name || login}</a>;
+  }
+
```

Comment:
good addition though... Could you please make a link color dull? `cause right now, It's not matching to our theme

Reply:
@abhisheksatre

Reply:
@NomanGul light black color added to match with the theme